addons/html_editor/static/src/main/chatgpt/chatgpt_dialog.js

```diff
@@ -84,9 +84,9 @@ export class ChatGPTDialog extends Component {
         let result = "";
         for (const child of fragment.children) {
             this.props.sanitize(child, { IN_PLACE: true });
-            result += child.outerHTML;
```

Comment:
Note: not a big deal (and might be detrimental for perf though it's probably irrelevant here)
But best to markup as close as possible to production (so outerHTML is, apparently, considered safe).
Blinding markuping the result at the end of a function not being a good idea (though it is case, it is so small that we can question it). Still much easier when reviewing security to see the markup on the correct line immediately.